### PR TITLE
Profile screen

### PR DESCRIPTION
**Pull request by fesqvw**

### CODE REVIEW

File: app/src/main/java/com/github/blecoeur/bootcamp/profile/ProfileActivity.kt [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        val statRecyclerView = findViewById<RecyclerView>(R.id.recyclerStats)
+        val statAdapter = StatViewAdapter( applicationContext , dbAccess.getStatsList(3, app.getActiveUser().ID))
+        statRecyclerView.adapter = statAdapter
+        statRecyclerView.layoutManager = LinearLayoutManager(applicationContext);
+
+        /* Games History */
+        val historyRecyclerView = findViewById<RecyclerView>(R.id.recyclerHist)
+        val historyAdapter = HistoryViewAdapter( applicationContext , dbAccess.getHistList(3,  app.getActiveUser().ID) )
+        historyRecyclerView.adapter = historyAdapter
+        historyRecyclerView.layoutManager = LinearLayoutManager(applicationContext);
```
Comment (Matthieu-Andre) [inline app/src/main/java/com/github/blecoeur/bootcamp/profile/ProfileActivity.kt:51]: There are some redundant semicolons on lines 39,45,51,57,63

File: app/src/main/java/com/github/blecoeur/bootcamp/profile/ProfileActivity.kt [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+    }
+
+    private fun activityStart(){
+        findViewById<ScrollView>(R.id.ProfileScroll).visibility = View.VISIBLE
+        findViewById<ScrollView>(R.id.InboxScroll).visibility = View.GONE
+        findViewById<ScrollView>(R.id.FriendsScroll).visibility = View.GONE
+    }
+
+    fun profileButton(view : View){
```
Comment (Matthieu-Andre) [inline app/src/main/java/com/github/blecoeur/bootcamp/profile/ProfileActivity.kt:78]: profileButton, inboxButton, friendsButton and settingButton all have a view parameter that is never used in theses methods

**Review by Matthieu-Andre — CHANGES_REQUESTED** [review]
Good job, code looks good.
Small nitpicks: there are some lines that have a redundant semicolon and some methods have parameters that they never use.
(also I think it's too much code for one pr, making it time intensive to review)

**Review by Matthieu-Andre — APPROVED** [review]
thanks for the fixes

**Review by dante11111111111 — APPROVED** [review]

File: app/src/main/java/com/github/blecoeur/bootcamp/profile/ProfileActivity.kt [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        /* Active user Information */
+        val app = applicationContext as MyApplication
+        dbAccess = app.getProfileDb()
+        findViewById<TextView>(R.id.profileUsername).text = app.getActiveUser().name
+
+        /* Achievements */
+        val achRecyclerView = findViewById<RecyclerView>(R.id.recyclerAch)
+        val achAdapter = AchViewAdapter( applicationContext , dbAccess.getAchList(3, app.getActiveUser().ID))
+        achRecyclerView.adapter = achAdapter
+        achRecyclerView.layoutManager = LinearLayoutManager(applicationContext)
```
Comment (Blecoeur) [inline app/src/main/java/com/github/blecoeur/bootcamp/profile/ProfileActivity.kt:39]: You could maybe replace it by a private function setup archRecyclerView

File: app/src/main/java/com/github/blecoeur/bootcamp/profile/ProfileActivity.kt [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+        /* Statistics */
+        val statRecyclerView = findViewById<RecyclerView>(R.id.recyclerStats)
+        val statAdapter = StatViewAdapter( applicationContext , dbAccess.getStatsList(3, app.getActiveUser().ID))
+        statRecyclerView.adapter = statAdapter
+        statRecyclerView.layoutManager = LinearLayoutManager(applicationContext)
```
Comment (Blecoeur) [inline app/src/main/java/com/github/blecoeur/bootcamp/profile/ProfileActivity.kt:45]: Same as archRecycleView

**Review by Blecoeur — APPROVED** [review]
Please avoid doing 57-files-changed PR,

**Pull request merged**